scipy/stats/tests/test_hypotests.py

```diff
@@ -1492,92 +1493,101 @@ def test_against_fisher_exact(self, alternative):
         assert_allclose(boschloo_stat, fisher_p)
 
 
+@make_xp_test_case(cramervonmises_2samp)
 class TestCvm_2samp:
     @pytest.mark.parametrize('args', [([], np.arange(5)),
                                       (np.arange(5), [1])])
-    def test_too_small_input(self, args):
-        with pytest.warns(SmallSampleWarning, match=too_small_1d_not_omit):
+    def test_too_small_input(self, args, xp):
+        args = (xp.asarray(arg, dtype=xp_default_dtype(xp)) for arg in args)
+        with eager_warns(SmallSampleWarning, match=too_small_1d_not_omit, xp=xp):
             res = cramervonmises_2samp(*args)
-            assert_equal(res.statistic, np.nan)
-            assert_equal(res.pvalue, np.nan)
+            xp_assert_equal(res.statistic, xp.asarray(np.nan))
+            xp_assert_equal(res.pvalue, xp.asarray(np.nan))
```

Comment:
Oops. Fortunately, this is safe to commit with `[skip ci]` if there is nothing else.
```suggestion
            xp_assert_equal(res.statistic, xp.asarray(xp.nan))
            xp_assert_equal(res.pvalue, xp.asarray(xp.nan))
```